src/ServiceControl/SagaAudit/ListSagasApi.cs

```diff
@@ -21,6 +22,8 @@ public class ListSagasApi : ScatterGatherApi<NoInput, List<SagaListIndex.Result>
                     .ToListAsync()
                     .ConfigureAwait(false);
 
+                results.ForEach(r => r.Uri = $"{r.Uri}?instance_id={instanceId}");
```

Comment:
Since ServiceInsight doesn't actually call this API, can we just remove it?

Reply:
ServicePulse does not use the list sagas API either. I vote we remove this API.

Reply:
I agree. If there are no official consumers of this API we should remove it.